build/patch/pre-autogen/libevent/common/000.patch

```diff
@@ -36,68 +13,62 @@ index ad815d9..babd653 100644
  pkgconfigdir=$(libdir)/pkgconfig
  LIBEVENT_PKGCONFIG=libevent.pc libevent_core.pc libevent_extra.pc
  
-@@ -122,12 +118,11 @@ endif
+@@ -130,7 +126,6 @@ endif
  
  if INSTALL_LIBEVENT
  lib_LTLIBRARIES = $(LIBEVENT_LIBS_LA)
 -pkgconfig_DATA = $(LIBEVENT_PKGCONFIG)
  else
  noinst_LTLIBRARIES =  $(LIBEVENT_LIBS_LA)
  endif
- 
--SUBDIRS = . include sample test
-+SUBDIRS = . include
+@@ -143,8 +138,6 @@ CLEANFILES=
+ DISTCLEANFILES=
+ BUILT_SOURCES =
+ include include/include.am
+-include sample/include.am
+-include test/include.am
  
  if BUILD_WIN32
  
 diff --git a/configure.ac b/configure.ac
-index 51725d6..86da55e 100644
+index 7528d37..24b65dc 100644
 --- a/configure.ac
 +++ b/configure.ac
-@@ -112,7 +112,6 @@ dnl Checks for libraries.
+@@ -121,10 +121,6 @@ AC_ARG_ENABLE([function-sections],
+ AC_ARG_ENABLE([verbose-debug],
+ 		AS_HELP_STRING([--enable-verbose-debug, verbose debug logging]),
+ 	[], [enable_verbose_debug=no])
+-AC_ARG_ENABLE([clock-gettime],
+-     AS_HELP_STRING(--disable-clock-gettime, do not use clock_gettime even if it is available),
+-  [], [enable_clock_gettime=yes])
+-
+ 
+ AC_PROG_LIBTOOL
+ 
+@@ -141,10 +137,6 @@ dnl Checks for libraries.
  AC_SEARCH_LIBS([inet_ntoa], [nsl])
  AC_SEARCH_LIBS([socket], [socket])
  AC_SEARCH_LIBS([inet_aton], [resolv])
--AC_SEARCH_LIBS([clock_gettime], [rt])
+-if test "x$enable_clock_gettime" = "xyes"; then
+-  AC_SEARCH_LIBS([clock_gettime], [rt])
+-  AC_CHECK_FUNCS([clock_gettime])
+-fi
```

Comment:
May you can leave this last part (from line 52 to 55). It should fail if you remove the part above (from line 41 to 43). This gives you a smaller diff. #bikeshedding